test/integration/targets/vyos_config/tests/cli/simple.yaml

```diff
@@ -25,6 +25,12 @@
     that:
       - "result.changed == false"
 
+- name: Delete services
+  vyos_config:
```

Comment:
```suggestion
  vyos_config: &del
```

Reply:
IMO it is not necessary. otherwise need to be updated for other as well.